src/cms/api/protected-area/services/protected-area.js

```diff
@@ -166,7 +171,9 @@ module.exports = {
         // and the description columns of park_activities and park_facilities
         // Any match here counts.
         query.where((builder) => {
-          builder.where(
+          builder.where("protected_areas.protectedAreaName", "ILIKE", `%${searchText}%`);
+          builder.orWhere("park_names.parkName", "ILIKE", `%${searchText}%`);
+          builder.orWhere(
```

Comment:
Instead of `LIKE` or `ILIKE`, it would be more better to use indexes to power these queries.  Currently the search is only handling left-handed anchored text and not true FTS /w partials.  `pg_trgm` can assist in this regard when using a GIN in pg.

e.g., `rrow` doesn't return any results.

In addition - getting a HTTP 500 anytime searching locally /w sqlite.  PR #222 is where this was introduced, not this particular commit.  That said, if we're going to abandon `SQLite` then we should spin up a ticket to remove it from our environment and force `pg` on everyone.

Reply:
Good points, I will look into `pg_trgm` or similar - I'd kind of ruled out using any extensions but it doesn't seem like it would be to complex.

Re: sqlite, that's an unfortunate side effect of using Postgres full text indexes as we do in search. Sqlite doesn't support them, and neither does knex (the db adapter library). I will create a ticket to remove references to sqlite as an option so that all devs are using Postgres.

Reply:
Updated with GIN indexes. Performance took a hit when making that change, with lots of queries well over 2 seconds, but I played around with it a bit and switched from one big JOIN/GROUP BY to a lot of subqueries and Postgres seems able to handle that better. It makes this change much bigger but I think it's an improvement.

Performance is still not ideal, seeing 1.5s or so for queries with a lot of results when I enable all parks (e.g. a query for "Park"). However most searches seem to be sub 500ms and "rrow" now brings up Arrow Lake Park. Also "garybaldi" brings up Garibaldi, etc.